en/config/actuators.md

```diff
@@ -79,6 +79,34 @@ The `X`, `Y`, `Z` positions are in [FRD coordinate frame, relative to the _centr
 Note, this may not be the same as the position of the flight controller!
 :::
 
+#### Geometry: Helicopter
+
+The geometry for a [Generic Helicopter (with Tail ESC)](../airframes/airframe_reference.md#copter_helicopter_generic_helicopter_(tail_esc)) is shown below.
+
+This is the only helicopter configuration supported, and allows for a single main rotor with any type of swash-plate and an ESC tail rotor.
+
+![Geometry: helicopter](../../assets/config/actuators/qgc_geometry_helicopter.png)
+
+The motors have no configurable geometry:
+
+- `Rotor (Motor 1)`: The main rotor
+- `Yaw tail motor (Motor 2)`: The tail rotor
+
+Swash plate servos:
+- Set the number of servos
```

Comment:
Looking at https://en.wikipedia.org/wiki/Swashplate_(aeronautics) you basically have to be able to tilt the plate to in any direction - which implies you need at least 3 servos, but that you could have more. Is there any "logic" here - i.e. will it really in practice tend to be just 3? What are PX4 limits?